ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.js

```diff
@@ -398,13 +408,16 @@
       netRegexJa: NetRegexes.dialog({ line: '月よ！\\s*鉄の覇道を照らせ！.*?', capture: false }),
       netRegexCn: NetRegexes.dialog({ line: '月光啊！\\s*照亮铁血霸道！.*?', capture: false }),
       netRegexKo: NetRegexes.dialog({ line: '달이여! 강철의 패도를 비춰라!.*?', capture: false }),
-      infoText: {
-        en: 'In => Out',
-        fr: 'Dedans => Dehors',
-        de: 'Rein => Raus',
-        ja: '密着 => 離れ',
-        cn: '靠近 => 远离',
-        ko: '안으로 => 밖으로',
+      infoText: (data, _, output) => output.text(),
```

Comment:
I'm going to weakly say no, because it's kind of weird to use a response for just one of these but not for all the rest.  In some hypothetical future if somebody changes the response inOut to use something other than `=>` then it'll look different than these?